test/gst-msdk/vpp/deinterlace.py

```diff
@@ -119,6 +126,14 @@ def before(self):
     *gen_vpp_deinterlace_parameters(
       spec_vc1, DeinterlaceTest._default_modes_))
   def test(self, case, method, rate):
-    vars(self).update(spec_vc1[case].copy())
-    vars(self).update(case = case, method = method, rate = rate)
+    self.init(spec_vc1, case, method, rate)
+    self.deinterlace()
+
+  @platform_tags(set(VC1_DECODE_PLATFORMS) & set(VPP_PLATFORMS))
+  @slash.requires(*have_gst_element("msdkvc1dec"))
+  @slash.parametrize(
+    *gen_vpp_deinterlace_parameters(
+      spec_vc1_r2r, DeinterlaceTest._default_modes_))
+  def test_r2r(self, case, method, rate):
+    self.init(spec_vc1_r2r, case, method, rate)
```

Comment:
missing `vars(self).setdefault("r2r", 5)`

Reply:
sorry, have fixed it.